homeassistant/components/foscam/__init__.py

```diff
@@ -1 +1,51 @@
 """The foscam component."""
+import asyncio
+
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+
+from .const import DOMAIN, SERVICE_PTZ
+
+PLATFORMS = ["camera"]
+
+CONFIG_SCHEMA = vol.Schema({DOMAIN: vol.Schema({})}, extra=vol.ALLOW_EXTRA)
```

Comment:
This config schema isn't used so we should remove it.

Reply:
This is not needed for `hassio.data[DOMAIN]`? Added it in [this](https://github.com/home-assistant/core/pull/41429/commits/f8630852f1d544a04d95005c707d9569fd1e5809) commit for tracking entities and being able to unload service when all the entities are unloaded.

Reply:
The config schema here is empty and we don't have any existing configuration under the foscam domain key, only under the camera domain key for the foscam camera platform.

So yes, we can remove this schema.

Reply:
New PR: https://github.com/home-assistant/core/pull/45154